specification/ml/_types/Detector.ts

```diff
@@ -30,6 +31,10 @@ export class Detector {
    * Custom rules enable you to customize the way detectors operate. For example, a rule may dictate conditions under which results should be skipped. Kibana refers to custom rules as job rules.
    */
   custom_rules?: DetectionRule[]
+  /**
+   * A description of the detector.
+   */
+  description?: string
```

Comment:
This situation is another strange one. On write we only accept `description`, but on read there's no `description` field but instead `datafeed_description`?